src/main/scala/intent/sbt/Runner.scala

```diff
@@ -83,12 +83,12 @@ class SbtTask(td: TaskDef, suit: IntentStructure, runner: TestSuiteRunner, focus
   override def taskDef = td
 
   // Tagging tests or suites are currently not supported
-  override def tags(): Array[String] = Array.empty
+  override def tags(): Array[String | Null] = Array.empty
```

Comment:
**Overridden java method returns array**
Overridden java method returns an array. Java arrays have type `Array[T | Null]`. Returned array must thereby be converted to have type `Array[T | Null]`.